data/ha/ensa2_crm_config.template

```diff
@@ -0,0 +1,38 @@
+primitive rsc_fs_%INSTANCE_SID%_ASCS%INSTANCE_ID_ASCS% Filesystem \
```

Comment:
Are comments supported in this kind of files? If it is so can you please add one line to explain what this fail is or to point the reader to some documentation?

Reply:
I am not sure. Haven't tried it. But I changed file name to ensa2_cluster_resources.template.